languages/ruby/examples/example.rb

```diff
@@ -0,0 +1,57 @@
+# NOTE - for example purpose only - import gem instead
+require 'bitwarden-sdk'
+
+token = ""
+
+bitwarden_settings = BitwardenSDK::BitwardenSettings.new(
+  'https://api.bitwarden.com',
+  'https://identity.bitwarden.com/connect/token'
```

Comment:
The URL here shouldn't need the path component
```suggestion
  'https://identity.bitwarden.com'
```